addons/markers/functions/fnc_onButtonClickConfirm.sqf

```diff
@@ -17,12 +17,34 @@
 params ["_buttonOk"];
 
 private _display = ctrlParent _buttonOk;
-private _description = _display displayctrl IDC_INSERT_MARKER;
+private _description = _display displayCtrl IDC_INSERT_MARKER;
 private _aceTimestamp = _display displayCtrl IDC_ACE_INSERT_MARKER_TIMESTAMP;
 
 // handle timestamp
 if (cbChecked _aceTimestamp && {ACE_player call FUNC(canTimestamp)}) then {
-    private _time = daytime;
+    private _time = [];
+
+    switch (GVAR(timestampTimezone)) do {
+        default {
+            _time = dayTime;
+        };
+        case 1: {
+            systemTime params ["", "", "", "_hour", "_min", "_sec"];
+            _time = _hour + _min/60 + _sec/3600;
+        };
+        case 2: {
+            _offset = GVAR(timestampUTCOffset);
+            systemTimeUTC params["", "", "", "_hour", "_min", "_sec"];
+            _hour = _hour + round(_offset);
+            if (_hour < 0) then {
+                _hour = 24 + _hour;
+            };
+            if (_hour > 24) then {
+                _hour = _hour - 24;
+            };
```

Comment:
SQF has modulo operator right? Could do 
```suggestion
            _hour = _hour % 24;
```

Reply:
Never used it with negative value's, but works great 👍